src/main/java/hng_java_boilerplate/email/EmailServices/EmailConsumerService.java

```diff
@@ -0,0 +1,27 @@
+package hng_java_boilerplate.email.EmailServices;
+
+import hng_java_boilerplate.email.entity.EmailMessage;
+import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.stereotype.Service;
+
+@Service
+public class EmailConsumerService {
+    private final JavaMailSender emailSender;
+
+    @Autowired
+    public EmailConsumerService(JavaMailSender emailSender) {
+        this.emailSender = emailSender;
+    }
+
+    @RabbitListener(queues = "emailQueue")
```

Comment:
Use the variable in the properties file.

Reply:
Done